concourse/scripts/common.sh

```
@@ -1037,12 +1082,13 @@ function run_eloqkv_tests() {
     local node_memory_limit_mb=8192
     local eloq_store_worker_num=2
     local eloq_store_data_path="/tmp/eloqkv_data/eloq_store"
-    local eloqkv_bin_path="/home/$current_user/workspace/eloqkv/cmake/eloqkv"
+    local eloqkv_bin_path="/home/$current_user/workspace/eloqkv/install/bin/eloqkv"
 
     # run redis with small ckpt interval.
     rm -rf ${eloq_data_path}/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)